src/cljc/rems/text.cljc

```diff
@@ -98,6 +100,23 @@
              (when time
                (format/unparse-local (time-format) (time/to-default-time-zone time))))))
 
+(defn localize-utc-date
```

Comment:
Would we actually want to have the server time-zone defined in the config for this? So that in Finland for Finnish officials you get more or less what you expect?

Reply:
Yeah something like that would be good to think about in the future. So far REMS has only referred to instants of time when something happened (event, entitlement), so storing those in UTC and rendering them in the browser timezone was the right thing to do.

Reply:
We decided to go with this and wait for the first installation that has a clear requirement about this.